libs/cardano-ledger-test/src/Test/Cardano/Ledger/Generic/PrettyCore.hs

```diff
@@ -2526,10 +2526,14 @@ instance PrettyA (VotingProcedure era) where
 
 -- ============================================================
 
-pcRewardAcnt :: RewardAcnt c -> PDoc
-pcRewardAcnt (RewardAcnt net cred) = ppSexp "RewAccnt" [pcNetwork net, pcCredential cred]
+pcRewardAccount :: RewardAccount c -> PDoc
+pcRewardAccount (RewardAccount net cred) = ppSexp "RewAccnt" [pcNetwork net, pcCredential cred]
 
-instance PrettyA (RewardAcnt c) where prettyA = pcRewardAcnt
+pcRewardAcnt :: RewardAccount c -> PDoc
+pcRewardAcnt = pcRewardAccount
+{-# DEPRECATED pcRewardAcnt "Use `pcRewardAccount` instead" #-}
```

Comment:
cardano-ledegr-test isn't one of the packages that we release, so no need to go through deprecation cycle
```suggestion
```